code/game/objects/explosion.dm

```diff
@@ -31,7 +31,7 @@
 		flame_range = min (MAX_EX_FLAME_RANGE, flame_range)
 
 	spawn(0)
-		var/start = TimeOfHour
+		var/watch = start_watch()

```

Comment:
timeofday...is really gross because it wraps around at midnight and becomes less precise later in the day.